docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,757 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Modules with prebuilt {anomaly-detect} jobs
+
+This page lists the available modules with prebuilt {anomaly-detect} jobs. Every 
+module item in the list contains a name, a short description, information about 
+the {ml-docs}/ml-buckets.html#ml-buckets[`bucket_span`] and the 
+{ml-docs}/ml-functions.html[analysis function] as well as the influencers – the 
+entities that have contributed to, or are to blame for, the anomalies – of the 
+{anomaly-detect} job.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
```

Comment:
Could we add something like this (moved out of https://www.elastic.co/guide/en/machine-learning/master/create-jobs.html)?:

> Context: These job wizards appear in Kibana if you use Filebeat to ship access logs from your Apache HTTP servers to Elasticsearch and store it using fields and datatypes from the Elastic Common Schema (ECS).